corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -220,7 +220,7 @@ hqDefine("data_dictionary/js/data_dictionary", [
                     }
 
                     _.each(group.properties(), function (element, index) {
-                        if (element.deleted && !element.id) {
+                        if (element.deleted() && !element.id) {
```

Comment:
:+1: 

This got caught in QA?

Reply:
This did. Creating case properties wasn't getting persisted on the back-end, which led to the discovery of this missing parenthesis. I think this bug slipped in when I was cleaning up the commit history and created a new PR.